doc/2/controllers/assets/replace-metadata/index.md

```diff
@@ -0,0 +1,80 @@
+---
+code: true
+type: page
+title: replaceMetadata
+description: Replace asset metadata
+---
+
+# replaceMetadata
+
+Replace `metadata` of an asset. It will replace only the fields specified in the request body.
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/_/device-manager/:engineId/assets/:_id/metadata
+Method: PATCH
+```
+
+### Other protocols
+
+```js
+{
+  "controller": "device-manager/assets",
+  "action": "replaceMetadata",
+  "engineId": "<engineId>",
+  "_id": "<assetId>",
+  "body": {
+    "metadata": {
+      "<metadata name>": "<metadata value>"
+    }
+  }
+}
+```
+
+## Arguments
+
+- `engineId`: Engine ID
+- `_id`: Asset ID
+
+## Body properties
+
+- `metadata`: Object containing metadata
+
+## Response
+
+```json
```

Comment:
Use the `js` label to better comment coloration (no comment when using `json`)
```suggestion
```js
```